src/main/java/io/debezium/connector/vitess/VitessConnectorConfig.java

```diff
@@ -248,6 +249,17 @@ public static BigIntUnsignedHandlingMode parse(String value, String defaultValue
                             + " If not configured, the connector streams changes from the latest position for the given shard(s)."
                             + " If snapshot.mode is INITIAL (default), the connector starts copying the tables for the given shard(s) first regardless of gtid value.");
 
+    public static final Field SHARD_EPOCH_MAP = Field.create(VITESS_CONFIG_GROUP_PREFIX + "shard.epoch.map")
```

Comment:
This is going to be a big map to pass in and potentially error-prone, when will people pass in this epoch on command line?   It this config mainly used for carry the current epoch values from a running system?